### DISCUSSION TITLE
como puedo ver los repositorios DONDE YO soy colaborador

### DISCUSSION THREAD

**Discussion by fernandoD91** [community/community#121542] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Buenas gente

quiero ver los repositorios donde me agregaron como colaborador pero no encuentro la forma.

me podrian decir por favor.

**Comment by ebndev** [comment]
Hi @fernandoD91, welcome!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Thank you for your understanding!
